### PR TITLE
Release 0.50

### PR DESCRIPTION
**Pull request by johanfylling**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,16 @@
 All notable changes to this project will be documented in this file. This
 project adheres to [Semantic Versioning](http://semver.org/).
 
+## 0.50.2
+
+This is a bug fix release that addresses a regression in 0.50.1.
```
Comment (srenatus) [inline CHANGELOG.md:8]: Perhaps we could include a simple example? It'll be easier to grasp what this is about, I think.

**Review by srenatus — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (johanfylling) [reply CHANGELOG.md:8]: Yep, I'll add one 👍 .

**Review by srenatus — APPROVED** [review]
`p` and `x` are our déformation professionnelle; but it's OK. I think I'd prefer the original example, maybe reduced a bit:

```rego
package example

import future.keywords

to_set(value) := value if {
	is_set(value)
} else = {elem | some elem in value} {
	is_array(value)
}
```

**Comment by johanfylling** [comment]
Even reduced, I think that example has a bit too much going on; running the risk of making it more difficult to grasp the issue, or to think that more components are required to trigger it.

**Pull request merged**